0022-circuit-block.md

```diff
@@ -272,21 +272,58 @@ class BackendV3:
 This might naturally extend to such a `BackendV3` providing the additional information that is needed for Qiskit SDK to safely transpile entire sampler and estimator pubs; the compilation of a pub _also_ implies further processing will be done by a quantum computer, just like an annotation remaining in a single circuit.
 
 
-## Case study: twirling, noise learning, and mitigation with `qiskit-ibm-runtime`
+## Case study: twirling, noise learning, and mitigation
```

Comment:
Maybe out of scope but could also talk about how PEA can be represented in this way? That is, emitting a Pauli error with rates multiplied by `num_amplifications`?

I remember there was some work @SamFerracin did where only a subset of entangling gates were amplified, this could be a way of specifying said subset (though deterministically as opposed to randomized here).